compass/default.cfg

```diff
@@ -28,6 +28,9 @@ verify = True
 # the program to use for graph partitioning
 partition_executable = gpmetis
 
+# the number of cores a user can use on a login node
+login_cores = 4
```

Comment:
Is it allowable to have this be more than 1 at most centers?

Reply:
Yes, this is thread parallelism and we're often allowed to use a few cores (e.g. you can compile code in parallel)